pkgs/by-name/kl/kleiner-brauhelfer-2/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  appimageTools,
+  qt6,
+  makeWrapper,
+}:
+let
+  pname = "kleiner-brauhelfer-2";
+  version = "2.6.2";
+
+  src =
+    fetchurl
+      {
+        x86_64-linux = {
+          url = "https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/download/v${version}/kleiner-brauhelfer-2-x86_64.AppImage";
+          hash = "sha256-PsIN7pg5vCrLlMD7Bm1HTmHK+RGyP10ifx842Kzy4H8=";
+        };
+      }
+      .${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242

---

removed the `with lib`